cookiecutter.json

```diff
@@ -1,15 +1,29 @@
 {
   "full_name": "First Middle Last",
   "email": "email@example.com",
-  "github_username": "<github username>",
   "autora_contribution_type": ["theorist", "experimentalist", "experiment_runner", "synthetic_data", "custom"],
+  "custom_autora_contribution_type": "{% if cookiecutter.autora_contribution_type == 'custom' -%}<custom_autora_contribution_type>{% else -%}N/A - Press Enter to Skip{% endif -%}",
+  "__autora_contribution_type": "{% if cookiecutter.autora_contribution_type == 'custom' -%}{{ cookiecutter.custom_autora_contribution_type }}{% else -%}{{ cookiecutter.autora_contribution_type }}{% endif -%}",  
+  "experimentalist_type": [
+    "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}sampler{% else -%}N/A - Press Enter to Skip{% endif -%}", 
+    "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}pooler{% else -%}N/A - Press Enter to Skip{% endif -%}", 
+    "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}custom{% else -%}N/A - Press Enter to Skip{% endif -%}"],
+  "custom_experimentalist_type": "{% if cookiecutter.experimentalist_type == 'custom' -%}<custom_experimentalist_type>{% else -%}N/A - Press Enter to Skip{% endif -%}",
+  "__experimentalist_type": "{% if cookiecutter.autora_contribution_type == 'experimentalist' and cookiecutter.experimentalist_type == 'custom' -%}{{ cookiecutter.custom_experimentalist_type }}{% else -%}{{ cookiecutter.experimentalist_type }}{% endif -%}",
+  "experiement_runner_type": [
+    "{% if cookiecutter.autora_contribution_type == 'experiment_runner' -%}experiment_runner{% else -%}N/A - Press Enter to Skip{% endif -%}", 
+    "{% if cookiecutter.autora_contribution_type == 'experiment_runner' -%}participation_manager{% else -%}N/A - Press Enter to Skip{% endif -%}", 
```

Comment:
```suggestion
    "{% if cookiecutter.autora_contribution_type == 'experiment_runner' -%}experimentation_manager{% else -%}N/A - Press Enter to Skip{% endif -%}", 
```

Reply:
is this right?

Reply:
No, but you are onto something. I was mixing recruitment_manager with participation_manager. There should be: recruitment_manage and experimentation_manager.

Reply:
Or... just looking at it again, you are right. I mixed those two up... Sorry for the confusion.

Reply:
ok, thought so! so globally we should have following, correct?

- Experiment Runner
    - Recruitment Manager
    - Experimentation Manager

Reply:
Yes.